readthedocs/projects/tasks.py

```diff
@@ -1296,12 +1309,16 @@ def build_docs_class(self, builder_class):
         builder.move()
         return success
 
-    def send_notifications(self, version_pk, build_pk, email=False):
-        """Send notifications on build failure."""
+    def send_notifications(self, version_pk, build_pk, event):
+        """Send notifications to all subscribers of `event`."""
         # Try to infer the version type if we can
         # before creating a task.
         if not self.version or self.version.type != EXTERNAL:
-            send_notifications.delay(version_pk, build_pk=build_pk, email=email)
+            build_tasks.send_build_notifications.delay(
```

Comment:
Actually, we may want to add this as a config for the Webhook. Like a checkbox for `Send notifications on PRs` so the user can decide whether or not to send them instead of us deciding by them.